### PR TITLE
[IOTDB-445] Unify the keyword of "timestamp" and "time"

### PR DESCRIPTION
**Pull request by Genius-pig**
use both timestamp and time.

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java [hunk 1]
```diff
@@ -1167,13 +1167,16 @@ private FilterOperator parsePredicate(PredicateContext ctx) {
...
-      BasicFunctionOperator basic = null;
+      BasicFunctionOperator basic;
       if (ctx.prefixPath() != null) {
         path = parsePrefixPath(ctx.prefixPath());
       }
       if (ctx.suffixPath() != null) {
         path = parseSuffixPath(ctx.suffixPath());
       }
+      if(ctx.TIME() != null || ctx.TIMESTAMP() != null) {
+        path = new Path(SQLConstant.RESERVED_TIME);
```
Comment (jixuan1989) [inline server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java]: I am not familiar with this part but there is one question, why there is no code about `path = new Path(SQLConstant.RESERVED_TIME);` in current master branch? (Or, why you add such a line of code?).
If no such codes, how current master branch support `select ..... where time > ...`?

Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java]: I guess the current master branch treats "time" as a time series path, it is not distinguished here

Comment (jixuan1989) [reply server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java]: OK.. As I am not familiar with this part. I do not know whether it is right... You can ignore my question when you decide whether to merge this pr.

**Comment by jixuan1989** [comment]
The failed  travis-ci/pr shows some potential bugs, though these bugs are not caused by this PR. 
I have downloaded the log.

**Pull request merged**